src/content/configuration/module.mdx

```diff
@@ -297,7 +297,8 @@ module.exports = {
           browserify: false, // disable special handling of Browserify bundles
           requireJs: false, // disable requirejs.*
           node: false, // disable __dirname, __filename, module, require.extensions, require.main, etc.
-          node: {...} // reconfigure [node](/configuration/node) layer on module level
+          node: {...}, // reconfigure [node](/configuration/node) layer on module level
+          worker: ["default from web-worker", "..."] // Customize the WebWorker handling for javascript files.
```

Comment:
Why is the "..." required?

Reply:
From tests 
https://github.com/webpack/webpack/blob/28580af775de8e166aed23189988199d4aa0c42e/test/configCases/worker/custom-worker/webpack.config.js#L11

Reply:
Yes i saw that PR, i dont understand what it does, this doc is cryptic to average reader if maintainers have no idea... I suggest we explain the `"..."` by asking relevant code contributors e.g. @sokra in slack before proceeding with the merge

Reply:
Thanks. I have added the following info about the `"..."`. 

https://github.com/webpack/webpack/pull/11531#issuecomment-700529559

Reply:
👍